zgw_consumers/models/validators.py

```diff
@@ -0,0 +1,58 @@
+from django.core.exceptions import ValidationError
+from django.core.validators import URLValidator
+from django.utils.deconstruct import deconstructible
+from django.utils.translation import gettext_lazy as _
+
+
+@deconstructible
+class PrefixValidator:
+    code = "invalid"
+
+    def __init__(
+        self,
+        prefix: str,
+        message: str = None,
+        code: str = None,
+        starts_with: bool = True,
+    ):
```

Comment:
this seems needlessy complex? The `PrefixValidator` is only used in one place, so the option for `starts_with` is not necessary and a bit over-engineered.

I also think you can replace this entire usage with the built in regex validator:

```py
>>> from django.core.validators import RegexValidator
>>> validator = RegexValidator(regex=r"^[^/#][^\s]*", message="The value must be a relative path.", code="invalid")
>>> validator("bar/baz")
>>> validator("/foo/bar/baz")
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/home/bbt/.virtualenvs/zgw-consumers/lib/python3.12/site-packages/django/core/validators.py", line 55, in __call__
    raise ValidationError(self.message, code=self.code, params={"value": value})
django.core.exceptions.ValidationError: ['The value must be a relative path.']
```

so you can get rid of this entire class :)